background/get-addon-settings.js

```diff
@@ -139,6 +139,24 @@ chrome.storage.sync.get([...ADDON_SETTINGS_KEYS, "addonsEnabled"], (storageItems
       }
     }
 
+    if (addonsEnabled["custom-menu-bar"] === undefined) {
+      // Transition v1.35 to v1.36
+      if (addonsEnabled["tutorials-button"] === true) {
+        // Hide Tutorials button is now a setting in Customizable menu bar. Enable it for existing addon users.
+        madeAnyChanges = true;
+        addonsEnabled["custom-menu-bar"] = true;
+        addonSettings["custom-menu-bar"] = { ["hide-tutorials-button"]: true };
```

Comment:
No idea why there's brackets [] here, but in this case it's equivalent as not having them.

Reply:
Double-checked the migration code, looks alright

Reply:
JavaScript doesn't really like hyphens in symbols.
```js
JSON.stringify({ a-b: 1 }); // > Uncaught SyntaxError: Unexpected token '-'
```
To work around this, brackets are needed.

Reply:
@DNin01 Sure, but the quotes are fixing that, not the brackets :P

Reply:
> @DNin01 Sure, but the quotes are fixing that, not the brackets :P

Oh.